jenkins.bash

```diff
@@ -81,9 +81,16 @@ EOF
 	then
 		EXTRA="EXTRA=${EXTRA}"
 	fi
+	if [[ "$version" = "v1.0" ]] && [[ "$CONTAINER" = "docker" ]] && [ $PYTHON_VERSION -eq 3 ]
+	then
+		tool_ver=master
```

Comment:
```suggestion
		tool_ver=$(cwltool --version | awk '{ print $2 }')
```

Reply:
How to specify the development version of cwltool?
The above change only works with the released version.

We sometimes want to know the result of conformance test for the latest (and not released) version.
It would be nice if `jenkins.bash` can store it to the directory such as `${implementation_name}/cwl_${cwl_version}/${implementation_name}_master/`.